crates/oxc_transformer/src/react_jsx/options.rs

```diff
@@ -67,13 +66,21 @@ impl Default for ReactJsxOptions {
     }
 }
 
-#[derive(Debug, Default, Clone, Copy, Deserialize)]
+#[derive(Deserialize, Debug, Clone)]
```

Comment:
The idea was stolen from https://github.com/serde-rs/serde/issues/912#issuecomment-1707349708, we want to catch four runtime input scenario, 
1. a valid input `automatic`
2. a valid input `classic`
3. None runtime.
4. wrong runtime that is neither `automatic` nor `classic`